### ISSUE TITLE
HTML not escaped on output

### ISSUE THREAD

**Issue by andybenedict** [gkushang/cucumber-html-reporter#52]
When the output report is created, the scenario and step names are not run through an HTML escape function and, as a result, any HTML that is part of a test scenario's name is executed.

This is primarily an issue for me when my team checks for injection on the site, meaning that I have to click my way through several dozen alerts when the report loads.

I don't work in javascript much, but I did a quick and dirty patch by adding an escapeHTML function to the features template, but I'm sure there is a much better and more comprehensive solution available.

```
<%
function escapeHtml(unsafe) {
    return unsafe
         .replace(/&/g, "&amp;")
         .replace(/</g, "&lt;")
         .replace(/>/g, "&gt;")
         .replace(/"/g, "&quot;")
         .replace(/'/g, "&#039;");
 }
%>
```

`<%= escapeHtml(step.name) %>`

I've noticed this on the bootstrap templates, but suspect that it affects the others, but I have not had the chance to test it.

Cheers,
Andy

**Comment by gkushang** [comment]
Let us test this out and add a support for it.

**Comment by hkhc** [comment]
may be just changing the `<%= step.name %>` to `<%- step.name %>` in templates?

**Comment by gkushang** [comment]
@hkhc's fix (https://github.com/gkushang/cucumber-html-reporter/pull/97) is published with  `cucumber-html-reporter@2.0.3`
